### ISSUE TITLE
Placement of buildings to create

### ISSUE THREAD

**Issue by Tropelage** [cepheidqq/FactorySkyline#10]
I can pick the buildings I want to create on the build, but I have no way to move them up or down. Is there a way to place, let's say a group of buildings above or to the left and up of where it tries to build it?

**Comment by cepheidqq** [comment]
I‘ve noticed this scenario for placing the building on the specific position. There is an inconvenient way to do that in the current version:
Step 1: Use the advanced copy and place the first copy on the ground
Step 2: Set position anywhere and change the displacement vector manually
Step 3: Place the second copy in the position you want, and dismantle the first one
It is a general method but is not convenient enough. The operation will be optimized in the future versions. And a fully enhanced position selecting tool is in the plan.
Maybe I will add a temporary solution of the up, down, left, right movement if the position tool take much more time.

**Comment by Tropelage** [comment]
How do I go about the second step? I will take a look at it and see if I can figure it out.

**Comment by cepheidqq** [comment]
A button called "Set Repeat" in the repeat copy panel

**Comment by Tropelage** [comment]
Hi cepheidqq,
Have you corrected the issue with placing the copies on a location and be able to set the height without the workaround you sent me last time? I know you updated the version, but I think the issue still persists.

Kind regards,

Oscar Capote

Sent with [ProtonMail](https://protonmail.com) Secure Email.

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐

**Comment by cepheidqq** [comment]
Yes, I'm working on it. It will release when the new input system finish.
